src/nativestore/PropertyLink.cpp

```diff
@@ -132,12 +162,16 @@ PropertyLink* PropertyLink::create(std::string name, char value[]) {
     PropertyLink::propertiesDB->write(dataName, PropertyLink::MAX_NAME_SIZE);
     PropertyLink::propertiesDB->write(reinterpret_cast<char*>(value), PropertyLink::MAX_VALUE_SIZE);
     if (!PropertyLink::propertiesDB->write(reinterpret_cast<char*>(&nextAddress), sizeof(nextAddress))) {
+        property_link_logger.log("XXXXXXXX : Error creating property link", "info");
```

Comment:
Delete this log line,

`property_link_logger.log("XXXXXXXX : Error creating property link", "info");`

Reply:
Done.

Reply:
Ok